opendocdb-cts/internal/testresult/test_result.go

```diff
@@ -12,7 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+// Package testresult provides functionality to format test suite results into a table.
+package testresult
```

Comment:
This package format results, but what generates those results? :thinking:

Reply:
The CTS generates those results. I've updated the comment